hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/ha/BootstrapStandby.java

```diff
@@ -254,6 +255,12 @@ private int doRun() throws IOException {
     return 0;
   }
 
+  @VisibleForTesting
+  public NamespaceInfo getProxyNamespaceInfo(NamenodeProtocol proxy)
```

Comment:
This needs to be used on line 184, or the test will not be able to mock the nsInfo object to make BootstrapStandby think it is actually a future version.